numpy/core/include/numpy/npy_cpu.h

```diff
@@ -115,4 +116,84 @@
 #define NPY_CPU_HAVE_UNALIGNED_ACCESS 0
 #endif
 
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+enum
```

Comment:
style nit: we don't mention enums in our style guide. We seem to have both the forms
```
enum NPY_MY_ENUM {
    ...
};
```

and 
```
typedef enum  {
    ...
} NPY_MY_ENUM;
```
It would be good to add a name to this enum using one of those.